### ISSUE TITLE
Tutorial for Widget TextEdit

### ISSUE THREAD

**Issue by s4Dt0y** [fogarecious/iced_tutorial#5]
I think that this one in particular would be a very useful tutorial. It took me a lot of pain to figure out how to get started with the [TextEdit](https://docs.rs/iced/latest/iced/widget/text_editor/struct.TextEditor.html) widget. I would open a pull request, but I have only just begun to get familiar with it...once I learn it completely I can open a PR. I'm not at that point yet, though. So I thought I'd open an issue and see if it gains any traction. Maybe I'll be able to open a PR by the end of this week.

**Comment by fogarecious** [comment]
Yes, welcome to add new materials. Basically, the tutorial is kept as simple as possible, so take it easy. Let me know if you need any help. Feel free to make a pull request and take your time.
